frontend/app/components/routing/wp-list/wp-list.controller.ts

```diff
@@ -159,6 +159,18 @@ function WorkPackagesListController($scope,
     AuthorisationService.initModelAuth("query", meta.query._links);
   }
 
+  $scope.setAnchorToNextPage = function () {
+    anchor = '';
+    if(jQuery('.pagination-number').last().hasClass('-current')) {
```

Comment:
Sorry to not have gotten back to you. I'll have a look whether we can add a more specific class to the pagination elements